src/view/items/shared/EventDirective.js

```diff
@@ -121,9 +122,14 @@ export default class EventDirective {
 
 			// Auto prevent and stop if return is explicitly false
 			let original;
-			if ( result === false && event && ( original = event.original ) ) {
-				original.preventDefault && original.preventDefault();
-				original.stopPropagation && original.stopPropagation();
+
+			if ( result === false ) {
+				if ( event && ( original = event.original ) ) {
+					original.preventDefault && original.preventDefault();
+					original.stopPropagation && original.stopPropagation();
+				} else {
+					warnIfDebug( 'Handler returns false, but no event object is available' );
+				}
```

Comment:
Consider putting `original` inside the block. It's only being used in there anyways. Also use `const` instead of `let` to indicate that it's never going to change.

I'm also not a fan of assignment inside conditions. It's hard to read and it's usually late to identify some value was retrieved then used.

```js
if(result === false){
  // Could have been null, but a missing property is undefined. ¯\_(ツ)_/¯
  const original = event ? event.original : undefined;

  if(original){
    original.preventDefault && original.preventDefault();
    original.stopPropagation && original.stopPropagation();
  } else {
    warnIfDebug( 'Handler returns false, but no event object is available' );
  }
}
```

alternatively... avoiding the additional indent.

```js

const falseReturn = result === false;
const original = event && event.original;

if(falseReturn && original){
  original.preventDefault && original.preventDefault();
  original.stopPropagation && original.stopPropagation();
} else if (falseReturn && !original){
  warnIfDebug( 'Handler returns false, but no event object is available' );
}
```

*By the way, these are just suggestions. The code is totally fine.* 😉